README.md

```diff
@@ -1,7 +1,7 @@
 # Safer custody with CTV vaults
 
 This repository demonstrates an implementation of simple, "single-hop" vaults
-using the proposed `OP_CHECKTEMPLATEVERIFY` opcode. 
+using the proposed `OP_CHECKTEMPLATEVERIFY` opcode.
```

Comment:
Would've preferred these doc/whitespace changes in a separate commit, but no big deal.

Reply:
Sorry, my editor did this and I know it sucks, but I thought it wasn't a very big deal.

I can rebase and remove these though.